test/gherkin/tests/test_get_policy.py

```diff
@@ -0,0 +1,37 @@
+import json
+import os
+from pytest_bdd import (
+    scenarios,
+    given,
+    when,
+    then,
+)
+
+scenarios("get_policy.feature")
+
+
+@given(
+    "an Agreement for MCC Registration exists",
+    target_fixture="mcc_registration_create"
+)
+def mcc_registration_create():
+    # Currently, this is just mocked in Caddy, so we don't create the object,
+    # we just return a hard-coded ID
+    return 1
```

Comment:
@taylordowns2000 

Notice that this simply shouldn't be done.

Yes, we can do something in the mock application that we rely on for a test. But the other examples won't be able to follow here. We need to actually create the objects that we expect to get back in later steps.

Reply:
This iteration its a mock.